server/repository/src/db_diesel/stocktake_line.rs

```diff
@@ -51,6 +52,11 @@ impl StocktakeLineFilter {
         self.item_id = Some(filter);
         self
     }
+
+    pub fn stock_line_id(mut self, filter: EqualFilter<String>) -> Self {
```

Comment:
At some point I did do something like this, but decided to err away from adding too much in repo layer in a patch